lib/Robo/Plugin/Commands/CleanCacheCommands.php

```diff
@@ -2,6 +2,10 @@
 
```

Comment:
Please add copyright statement at the top of the file, before namespace.

Reply:
Added